content/docs/command-reference/plots/index.md

```diff
@@ -158,6 +158,23 @@ header (first row) are equivalent to field names.
 
 - `<DVC_METRIC_Y_TITLE>` (optional) - field name to display as the X axis label
 
+## HTML templates
+
+By default, the plots generated by `dvc plots` require Internet access to load
+Vega JavaScript libraries. It's possible to supply a custom HTML file to the
+`--html-template` option of `dvc plot show` and `dvc plot diff` when the
+Internet access is not available.
```

Comment:
We shouldn't present this as a solution for offline envs initially. You can also use this feature even when you're online. Let's move most of that motivation to the example instead please.

Reply:
So what's a more general motivation for this section that explains the feature? Perhaps something like "The default plots use a plain HTML file which uses cloud Vega libraries." Plus maybe link to the official "Vega libs" location. Is it https://github.com/vega/vega/releases?

Reply:
Another motivation may be to put some extra stuff to the plots, like a banner, title etc. But in general, I think people will understand if something works _offline_, it can also work _online._

Reply:
Yes offline vs online is not the most general use, right? Templates are about customization in general. Offline js imports just happens to be one thing you can customize.